src/tasks/installers/olm.ts

```diff
@@ -136,6 +136,17 @@ export class OLMTasks {
           task.title = `${task.title}...done.`
         }
       },
+      {
+        title: 'Change operator image',
+        enabled: () => !!flags['che-operator-image'],
+        task: async (_ctx: any, task: any) => {
+          const csvList = await kube.getClusterServiceVersions(flags.chenamespace)
+          const csv = csvList.items[0]
```

Comment:
if `items` is empty?

Reply:
I don't think they can be empty, because we `waitCatalogSource` above.
But will add a check just for case. Anyway we cannot fix it, if the list is empty.